doc/api/fs.markdown

```diff
@@ -577,7 +577,11 @@ If you want to be notified when the file was modified, not just accessed
 you need to compare `curr.mtime` and `prev.mtime`.
 
 _Note: when an `fs.watchFile` operation results in an `ENOENT` error, it will
-invoke the callback once. This is a change in functionality since v0.10._
+ invoke the listener once, with two stat objects. All the fields in those stat
+ objects will be set to zero, except for the time fields, which are set to
+ Thu Jan 01 1970 00:00:00. If the file got created later on, the listener will
```

Comment:
`s/got created/was created`? "got" is more colloquial and I'd say its better to use the alternative in docs.